lapis/storage_io/storage.py

```diff
@@ -21,7 +21,7 @@ def storage_reader(
             name=row["name"],
             sitename=row["sitename"],
             size=int(
-                row["cachesizeGB"] * unit_conversion_mapping.get("cachesizeGB", 1)
+                int(row["cachesizeGB"]) * unit_conversion_mapping.get("cachesizeGB", 1)
```

Comment:
@eileen-kuehn I hope that this incorporates what we both wanted it to do. Please consider that without typecasting `row["cachesizeGB"]` is a string and `row["cachesizeGB"]  * 1024 * 1024 * 1024` leads to a very long string and a very full RAM and sometimes, at least in my case, a crashing computer ;)